### ISSUE TITLE
Go to file keyboard shortcut triggers bad behavior in webview

### ISSUE THREAD

**Issue by RasmusWL** [microsoft/vscode#88284]
When I have a webview window active (such as previewing a markdown file), and I open _Go to file_ menu with `Ctrl+P`, when I release `Ctrl` it will directly open the currently selected file :open_mouth: 

This does not happen when I open the _Command Palette_ with `Ctrl+Shift+P`. 

VS Code version: Code 1.41.1 (26076a4de974ead31f97692a0d32f90d735645c0, 2019-12-18T15:04:31.999Z)
OS version: Linux x64 5.0.0-37-generic

Related issues (but not exactly this case)
- https://github.com/microsoft/vscode/issues/85879
- https://github.com/microsoft/vscode/issues/82592
- https://github.com/microsoft/vscode/issues/75421

Steps to Reproduce:

1. Open a webview
2. Open _Go to file_ menu with `Ctrl+P`
3. Once you release `Ctrl` the currently selected file will be opened

Does this issue occur when all extensions are disabled?: N/A -- no clue how to launch a webview without my extensions :smile:

**Comment by mjbvz** [comment]
I can't repo this. @RasmusWL 

Does this reproduce in the [latest VS Code insiders build](https://code.visualstudio.com/insiders/)?

Also, be sure to test using one of the built-in webview pages, such as the release notes or the markdown preview

**Comment by RasmusWL** [comment]
@mjbvz yes this reproduces on my Ubuntu 18.04 machine in the latest VS Code insiders build. 

Keyboard shortcut for opening new file (Ctrl+N) also opens _two_ new files, and keyboard shortcut for closing files (Ctrl+W) closes _two_ files :confused: 

<details>
<summary>Full version details</summary>

```
Version: 1.45.0-insider
Commit: 6e9a34c0de4181a99f11e106733ee3e8c9db79f1
Date: 2020-04-16T05:44:19.773Z
Electron: 7.2.1
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Linux x64 5.3.0-46-generic
```
</details>

**Comment by mjbvz** [comment]
Thanks for checking. Fixed by dbc7b4d912d34da083c969e5ee921f74119c30b9 for me

Let me know if you still see this in the next VS Code insiders build

**Comment by RasmusWL** [comment]
Fixed in latest insiders build :tada: :heart:
